YuriiDatsiuk/HW11/11_1.py

```diff
@@ -0,0 +1,23 @@
+
+
```

Comment:
good job
look at this version of your code:
def check_age(age):
    try:
        age = int(age)
    except ValueError as e:
        print('It\'s not an integer. Please enter a valid age.', e)
        return

    if age < 0:
        raise ValueError('Entered a negative number')
    else:
        if age % 2 == 0:
            print('Age is even')
        else:
            print('Age is odd')


try:
    age = input('Enter your age: ')
    check_age(age)
except ValueError as ve:
    print(f'Error: {ve}')
except Exception as e:
    print(f'An unexpected error occurred: {e}')